src/solvers/refinement/string_builtin_function.cpp

```diff
@@ -187,6 +187,25 @@ exprt string_set_char_builtin_functiont::length_constraint() const
     equal_exprt(return_code, return_value));
 }
 
+optionalt<exprt> string_to_lower_case_builtin_functiont::eval(
+  const std::function<exprt(const exprt &)> &get_value) const
+{
+  auto input_opt = eval_string(input, get_value);
```

Comment:
Not related to this PR, but `eval_string` could benefit from some documentation.